loopchain/utils/command_arguments/__init__.py

```diff
@@ -65,7 +65,7 @@ def command(self, value):
 
 attributes = {
     Type.ServiceType:
-        Attribute("service_type", type=str,
+        Attribute("service_type", type=str, default='citizen', nargs='?',
                   help="loopchain service to start [rs|peer|citizen|rest|rest-rs|score|tool|admin]"),
```

Comment:
Are we use all of NodeTypes written in help msg?
If not, it is better to rewrite message.

Reply:
You're right. `rs, rest rest-rs` won't be needed anymore.